docs/10_domains/60_electrochemistry.mdx

```diff
@@ -0,0 +1,67 @@
+<LbeChip title="physical chemistry" />
+<LbeChip title="electrochemistry" />
+
+## Introduction
+
+Electrochemistry plays a significant role in very different research areas, such as organic chemistry, analytical chemistry, life sciences, engineering and material science.
+Research is therefore often highly interdisciplinary, and the number of techniques and measurement procedures available are vast.
+
+## Data Types
+
+Conceptually, the main data type in electrochemistry consists of time series data containing fundamental electrical parameters such as voltage (potential)
+and current (current density), including the measurement time.
+For AC methods also the frequency, the impedance and the phase angle are relevant.
+These data are most simply stored as CSV, but are often only available in non-standard file formats.
+Often additional techniques are employed *operando* to gain additional insight into the measured charge transfer
+and other (chemical) processes at the electrodes and the electrolyte,
+such as IR-spectroscopy, mass spectrometry or diffraction methods,
+which are stored along with the values from the electrochemical measurements.
+
+Additionally, imaging data can be obtained from purely electrochemical techniques such as scanning electrochemical microscopy (SECM) or indirectly,
+when combined with other methods such as *in situ* electrochemical scanning tunneling microscopy (*in situ* STM).
+Further data evaluation can yield any kind of time series data or imaging data.
+
+## ELNs and Other Tools
+
+
+For effective data management, software tools should be selected in a uniform manner within a project or research group with the aim to [organize](/docs/data_organisation)
+and streamline workflows. This involves establishing clear usage guidelines, including metadata templates drawn from minimum information standards for a given method, where available.
+These should be outlined in a [data management plan (DMP)](/docs/dmp) for each project. Many universities supply tools and templates for DMPs (see the [respective article](/docs/dmp) for more information).
+
+An [electronic lab notebooks (ELN)](/docs/eln) helps in the day-to-day planning and structured documentation of experiments,
+while some also assist in data workflow management.
+For electrochemical data, ELNs must be flexible and customizable, to meet the requirements of the diverse research topics.
+As such, there exists to date no ELN that meets the requirements of electrochemistry alone and researchers are may choose an ELN that fits best their research scope.
+The [ELN-Finder](https://eln-finder.ulb.tu-darmstadt.de/search?f.K03=Pharmacy,equals&spc.page=1)
+lists many options and the article on [choosing an ELN](/docs/choose_eln) provides further assistance.
+General purpose ELNs can be found with the keyword "Physical chemistry":
+
+<ElnFinder subDisc="Physical chemistry" />
+
+For specific domains, such as "electrosynthesis" the Chemotion ELN might be an appropriate choice.
```

Comment:
Chemotion ELN should be a link.
Not sure if you want to link to the Chemotion ELN homepage or on a subpage in the knowledge base